firefox-ios/firefox-ios-tests/Tests/ClientTests/TabWebViewPreviewStateTests.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Redux
+import Common
+
+@testable import Client
+
+final class TabWebViewPreviewStateTests: XCTestCase, StoreTestUtility {
+    override func setUp() {
+        super.setUp()
+        setupStore()
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
we should move the parent tearDown to the bottom. Reference: https://github.com/mozilla-mobile/firefox-ios/wiki/Unit-Tests-Guidelines#1-setupteardown